andela/boondocks-bn-backend
# #169257385 Configure ESLint on project structure

#### What does this PR do?
This pull request sets up ESLint for this project to enforce a common coding style

#### Description of Task to be completed
- [x] install ESLint and its dependencies
- [x] create an ESLint configuration file

#### How should this be manually tested?
- After cloning the repo 'cd' into it
- run 'npm install' to install all the dependencies
- open the project folder using an editor of your choice and you should see the following:
  - an 'eslintrc.json' file whose rules match those provided in the relevant PT board story (see below)
  - the 'package.json' file will contain all the required ESLint installations under 'devDependencies'

#### Any background context you want to provide?
This ESLint configuration will allow the team to produce a standardized code base that is clean and easy to understand

#### What are the relevant pivotal tracker stories?
[#169257385](https://www.pivotaltracker.com/story/show/169257385)

#### Screenshots (if appropriate)
N/A
